### PR TITLE
bpo-45925: Update macOS installer to SQLite 3.37.2

### PR DESCRIPTION
**Pull request by kumaraditya303**
https://bugs.python.org/issue45925

### CODE REVIEW

**Review by erlend-aasland — APPROVED** [review]
LGTM. Please fix PR title: _SQlite_ => _SQLite_ ;)

File: Misc/NEWS.d/next/Build/2022-01-26-12-04-09.bpo-45925.yBSiYO.rst [hunk 1]
```diff
@@ -0,0 +1 @@
+Update macOS installer to SQlite 3.37.2.
```
Comment (erlend-aasland) [inline Misc/NEWS.d/next/Build/2022-01-26-12-04-09.bpo-45925.yBSiYO.rst]:
```suggestion
Update macOS installer to SQLite 3.37.2.
```

**Comment by ned-deily** [comment]
@AlexWaygood I'm working on the backports.

**Comment by AlexWaygood** [comment]
Yes, sorry, I misread miss-islington's message.

**Pull request merged**